ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -186,7 +188,7 @@ Localization {
     #Principia_FlightPlan_StatusMessage_ChangeFlightPlan = 请更换轨道规划
     #Principia_FlightPlan_StatusMessage_FailedError = 计算失败，出现错误 <<1>> 错误信息 \"<<2>>\"  // <<1>> status_.error <<2>> status_.message
     #Principia_FlightPlan_StatusMessage_TimeOut = 在<<1>>之后  // <<1>> Timeout
-    #Principia_FlightPlan_StatusMessage_MaxSteps = 积分器已达到最大步数<<1>>  // <<1>> Timeout
+    #Principia_FlightPlan_StatusMessage_MaxSteps = 积分器已<<1>>达到最大步数  // <<1>> Timeout
     #Principia_FlightPlan_StatusMessage_MaxSegment = 请增加“最大迭代步长”或避免与天体碰撞
     #Principia_FlightPlan_StatusMessage_Singularity = 积分器遇到奇点（可能是天体的中心）<<1>>  // <<1>> timeout
```

Comment:
The existing string talks about the time taken by the computation, not the time of the singularity; Zethar recommends:
> 积分器在算+<<1>>时刻时遇到了奇点